scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -205,7 +207,20 @@ class FossId(
         val (results, duration) = measureTimedValue {
             val results = mutableMapOf<Package, MutableList<ScanResult>>()
 
-            packages.forEach { pkg: Package ->
+            log.info {
+                if (packageNamespaceFilter.isEmpty()) "No package namespace filter is set."
+                else "Package namespace filter is '$packageNamespaceFilter'."
+            }
+
+            val filteredPackages = packages
+                .filter { p -> packageNamespaceFilter.isEmpty() || p.id.namespace == packageNamespaceFilter }
+
+            if (filteredPackages.isEmpty()) {
+                log.warn { "There is no package to scan !" }
+                return results
+            }
+
+            filteredPackages.forEach { pkg: Package ->
```

Comment:
I realize it was like that before, but you may want to take the chance to omit the explicit `: Package` type.